File: src/main/java/com/xue/Hello.java
package com.xue;

public class Hello {
    public static void main(String[] args) {
        //  int 32bit short 16bit long 64bit byte 8bit
        //  char '字'
        //  float 32bit double 64bit
        //        new Person().hello();
        //  float weight = 66.5f; 如不加f/F會錯誤 因java看到小數點會自動以double儲存 故在數字後面加f
        Person p = new Person();
        p.hello();
        int age = 25;
        char c = '哈';
        byte b = 120;
        float d = 66.5f;
        double e = 99.9;
        boolean adult = true;
        String abc = "1234";
    }
}

File: src/main/kotlin/com/xue/Hello.kt
package com.xue

fun main(args: Array<String>) {
//    println("Hello")
//    Human() --> 以產生物件
//    Human().hello() //但使用完就消失
//    var 可變 val不可變
    val h = Human()//val 代表定義完這物件就不能更改他的值

    h.hello()
    var age: Int = 19 // var age=19 會自動推斷
    age = 20
//    如建立String 之後再給值可做以下建立
    var name: String
    name = "HANK"

}

class Human {
    fun hello() {
        println("Hello Kotlin")

    }
}